sources/packages/backend/apps/queue-consumers/src/processors/schedulers/cra-integration/_tests_/cra-response-integration.scheduler.e2e-spec.ts

```diff
@@ -0,0 +1,116 @@
+import { createMock, DeepMocked } from "@golevelup/ts-jest";
+import { INestApplication } from "@nestjs/common";
+import { QueueNames } from "@sims/utilities";
+import {
+  createTestingAppModule,
+  describeProcessorRootTest,
+} from "../../../../../test/helpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeCRAIncomeVerification,
+  saveFakeApplication,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import * as Client from "ssh2-sftp-client";
+import * as path from "path";
+import { CRAResponseIntegrationScheduler } from "../cra-response-integration.scheduler";
+import {
+  createFileFromStructuredRecords,
+  getStructuredRecords,
+  mockDownloadFiles,
+} from "@sims/test-utils/mocks";
+import { Job } from "bull";
+import { ApplicationStatus } from "@sims/sims-db";
+
+const CRA_FILENAME = "CRA_200_PBCSA00000.TXT";
+
+const padWithLeadingZeros = (num: number): string => {
+  // Pad the number with leading zeros to make it 9 digits long
+  return num.toString().padStart(9, "0");
+};
```

Comment:
Helper methods are usually created at the end of the tests, inside the `describe` they belong.
For this one, I would refrain from creating the method, but keeping it will no be a blocker if moved to the right place and added method comments.

Reply:
The PR does not seem to be updated with the change.

![image](https://github.com/user-attachments/assets/5733c28a-1409-44b9-a4f6-61c30becebd0)

Please read the comment given carefully to ensure that nothing is missing.